stable/redis/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 3.0.5
+version: 3.0.6
```

Comment:
This adds a new feature, please bump the minor version instead of the patch level.

Reply:
so version is 3.1.0?

Reply:
Indeed.